data/modules/scripts/gamestore/init.lua

```diff
@@ -1743,7 +1743,7 @@ function GameStore.processExpBoostPuchase(player)
 		player:setStorageValue(GameStore.Storages.expBoostCount, 1)
 	end
 
-	player:setStorageValue(GameStore.Storages.expBoostCount, expBoostCount + 1)
+	player:setStorageValue(GameStore.Storages.expBoostCount, player:getStorageValue(GameStore.Storages.expBoostCount) + 1)
```

Comment:
on the line 1737 we have this variable
`local expBoostCount = player:getStorageValue(GameStore.Storages.expBoostCount)`
is the same thing to use `expBoostCount`